.github/workflows/build-and-push-binaries.yml

```
@@ -27,6 +27,30 @@ env:
       { "os": "windows-latest", "shell": "cmd" }
     ]
 jobs:
+  # Download cross-platform Java dependencies (bundle.jar, maven.default.index)
+  # from the jdtls-server-base container image. These are platform-independent
+  # and get bundled into the release zips for the editor-extensions to extract.
+  download_java_deps:
+    name: Download Java analysis dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)